src/MobileStepper/MobileStepper.js

```diff
@@ -0,0 +1,165 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { createStyleSheet } from 'jss-theme-reactor';
+import withStyles from '../styles/withStyles';
+import Paper from '../Paper';
+import Button from '../Button';
+import KeyboardArrowLeft from '../svg-icons/keyboard-arrow-left';
+import KeyboardArrowRight from '../svg-icons/keyboard-arrow-right';
+import { LinearProgress } from '../Progress';
+
+export const styleSheet = createStyleSheet('MuiMobileStepper', theme => ({
+  mobileStepper: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    width: '100%',
+    position: 'fixed',
+    bottom: 0,
+    left: 0,
+    zIndex: theme.zIndex.mobileStepper,
+    backgroundColor: theme.palette.background.paper,
+    padding: '6px',
+  },
+  button: {},
+  dots: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  dot: {
+    backgroundColor: theme.palette.action.disabled,
+    borderRadius: '50%',
+    width: '10px',
+    height: '10px',
+    margin: '0 2px',
+  },
+  dotActive: {
+    backgroundColor: theme.palette.primary[500],
+  },
+  progress: {
+    width: '50%',
+  },
+}));
+
+function MobileStepper(props) {
+  const {
+    activeStep,
+    buttonClassName: buttonClassNameProp,
```

Comment:
I don't think that we need that property as already accessible with `classes.button`.